bears/general/SpaceConsistencyBear.py

```diff
@@ -38,7 +41,47 @@ def run(self,
         result_texts = []
         additional_info_texts = []
 
-        for line_number, line in enumerate(file, start=1):
+        def get_blanklines_nr_start():
+            line_nr_start = False
+            rev_enumerated_zip_obj = zip(reversed(range(1, len(file) + 1)),
+                                         reversed(file))
+            for line_number, line in rev_enumerated_zip_obj:
+                replacement = line
+                if replacement.strip() == '':
+                    line_nr_start = line_number
+                else:
+                    break
+
+            return line_nr_start
+
+        if allow_trailing_blanklines:
+            end_line_of_file = len(file)
```

Comment:
what about `blanklines_start_line`? I think that would be a good name :)
(yeah I finally got a name for it :D)